modules/gke-cluster-standard/main.tf

```diff
@@ -149,6 +149,25 @@ resource "google_container_cluster" "cluster" {
               enable_secure_boot          = var.cluster_autoscaling.auto_provisioning_defaults.shielded_instance_config.secure_boot
             }
           }
+          upgrade_settings {
```

Comment:
This entire block should be dynamic and conditional on `upgrade_settings` being defined.

Reply:
Also considering using iterator (or the default block name) to make the nested blocks easier to read